content/ch-quantum-hardware/Jaynes-Cummings-model.ipynb

```diff
@@ -66,36 +66,8 @@
     "from qiskit.compiler import transpile, assemble\n",
```

Comment:
<p>I don't see any reason to use level1 measurement. The signal outcome may be sometime inverted depending on the calibration of the meas pulse. I'm also surprised that qubit T1 is really high.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>